Woof/WoofTests/Model/LocalStorage/LocalStorageDeleteTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+
+final class LocalStorageDeleteTests: XCTestCase {
+    var localStorage: LocalStorage!
+
+    override func setUp() {
+        let storage = LocalStorage()
+        localStorage = storage
+
+        UserDefaults.standard.removePersistentDomain(forName: Bundle.main.bundleIdentifier!)
```

Comment:
Please do not use force unwrap.
Have you run linters? Does the linter show the corresponding warning?

Reply:
Removed as unnecessary.